### DISCUSSION TITLE
Updated version 0.1 is not starting jobs

### DISCUSSION THREAD

**Discussion by Hammer8** [swar/Swar-Chia-Plot-Manager#630] [General]
Hi, updated to new version and it doesn't seem to kick off new jobs...maybe there's a special way to upgrade?  I rebooted, extracted files to new director and did python manager.py start.  It starts fine, but no jobs are kicked off.  I checked in process manager as well just in case...

Also, it was throwing off error trying to get disk space for None.  Is that due to my using a network path for destination?  I mapped it to a drive letter and set skip_full_destinations: false and that seems to fix the issue.

Thanks!!

**Comment by tritechnik** [comment]
Mapped drive letters don't work use UNC paths.` \\SERVER\Share`

**Reply by Hammer8** [reply]
Thanks...that did the trick although I still need to turn off option to check disk space.  Thank you!
